daemon/src/maker_inc_connections.rs

```diff
@@ -182,7 +182,7 @@ impl Actor {
         heartbeat_interval: Duration,
         p2p_socket: SocketAddr,
     ) -> Self {
-        NUM_CONNECTIONS_GAUGE.set(0);
+        NUM_CONNECTIONS_GAUGE.reset();
```

Comment:
❓ This is equivalent?

Reply:
As far as I know, yes.

https://docs.rs/prometheus/0.13.0/prometheus/core/struct.MetricVec.html#method.reset